tests/test_tablepyxl.py

```diff
@@ -118,6 +125,12 @@ def test_document_to_workbook(self):
         wb = document_to_workbook(table_three, wb=wb)
         self.assertEqual(wb.sheetnames, ['simple table', 'second table', 'Another simple table'])
 
+    def test_comments(self):
+        wb = document_to_workbook(table_comment)
+        sheet = wb['comment table']
+        self.assertNotEqual(sheet['A1'].value, 'this is a html comment')
```

Comment:
Instead of asserting that it's not equal, how about asserting it doesn't contain the same text? Before this code change, I would expect the value could be `'<!-- this is a html comment -->'` and your test would still pass.